src/containers/Places/Place.tsx

```diff
@@ -38,7 +38,11 @@ function Place() {
   useEffect(() => {
     if (place.data) {
       sensorsActions.onRequest();
-      getSensors(place.data, sensorsActions.onSuccess, sensorsActions.onError);
+      getSensorsInPlace(
```

Comment:
Bit of a nit (no need to change it) but this functions gets sensors by place not in place

Reply:
Done!